CHANGELOG.md

```diff
@@ -4,6 +4,23 @@
 
 - "You miss 100 percent of the chances you don't take. — Wayne Gretzky" — Michael Scott
 
+## 7.29.0
+
+This update includes a change to the `@sentry/nextjs` SDK that may increase response times of requests in applications
+deployed to Vercel or AWS lambdas to ensure that error events are sent consistently.
+Additionally, Next.js applications deployed to Vercel or AWS lambdas may also see an uptick in sent transactions. (for
+more information see #6578)
+
+- feat(core): Add `getSdkMetadata` to `Client` (#6643)
+- feat(nextjs): Send events consistently on platforms that don't support streaming (#6578)
+- feat(replay): Use new `prepareEvent` util & ensure dropping replays works (#6522)
+- feat(types): Upstream some replay types (#6506)
+- fix(replay): Envelope send should be awaited in try/catch (#6625)
+- fix(replay): Improve handling of `maskAllText` selector (#6637)
+- fix(tracing): Don't finish React Router 6 `pageload` transactions early (#6609)
+- ref(core): Expose `prepareEvent` util function (#6517)
```

Comment:
Not sure, but I think we can omit these last two changes as they are purely internal? WDYT?

Reply:
Yeah totally. I removed them.